solr/modules/hdfs/src/test/org/apache/solr/store/blockcache/BlockCacheTest.java

```diff
@@ -257,7 +259,7 @@ public void testCacheConcurrent() throws Exception {
           return;
         }
       }
-      assertEquals("cache key differs from value's key", k, (Long) v.key);
+      Assert.assertEquals("cache key differs from value's key", k, (Long) v.key);
```

Comment:
minor: you could just add static imports for this stuff.  I'm not sure why these changes are happening.

Reply:
This can be cleaned up. Will push a fix shortly. Looks like its only the `test/org/apache/solr/store` tests

Reply:
Fixed in e75d6ac1bc6c6ee7777f23ea0b2b8877928b86e6